clusters/hive/pools/msp/msp-ocp-4-11-medium-single-az-aws_clusterpool.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       region: us-east-1
   pullSecretRef:
     name: pull-secret
-  size: 4
+  size: 0
```

Comment:
Shouldn't this be `1`?

Reply:
No, our intention is to significantly reduce the resources on AWS for this project (MSP). Our priorities and plans were changed and there is no need to keep any hibernated cluster in MSP pools.